addons/web_editor/static/src/js/common/ace-dialog.js

```diff
@@ -0,0 +1,109 @@
+/** @odoo-module **/
```

Comment:
Commit message convention:
- Some info here: https://www.odoo.com/documentation/16.0/contributing/development/git_guidelines.html#commit-message-header
- Max 72 characters per line (title and message) (some editors actually set up a 72 chars ruler automatically, you could set one)
- No point at the end of the commit title
- The title must complete the sentence "when applied, this commit will ..."
- You commit should mention all the apps you touched (so not only web_editor here)